tools/bin/ariaDownload.py

```diff
@@ -211,8 +214,11 @@ def make_nc_cookie(self):
            user  = self.inps.user
            passw = self.inps.passw
        elif op.exists(op.join(op.expanduser('~'), '.netrc')):
-           # log.info('Obtaining user/pass from .netrc')
-           user, _,  passw = netrc.netrc().authenticators('urs.earthdata.nasa.gov')
+           log.info('Attempting to obtaining user/pass from .netrc')
+           try:
+               user, _,  passw = netrc.netrc().authenticators('urs.earthdata.nasa.gov')
+           except:
```

Comment:
Codacy found an issue: [No exception type(s) specified](https://app.codacy.com/gh/aria-tools/ARIA-tools/pullRequest?prid=5957665)

Reply:
@Askaholic any thoughts?

Reply:
Either specify the type of exception you are expecting there (AuthenticationError, ConnectionError, OSError, etc) or use `except Exception:` for a "catchall" exception handler. Generally writing "catchall" exception handlers is discouraged as you can usually do better, unless you are in the situation where you really aren't expecting any exceptions to be thrown, but you're writing a handler just in cases you missed something and can report it for future bugfixing later.

Reply:
As the latter is not the case in this situation, I would advise adding a few `except` clauses to catch common errors you expect, and print a helpful error message. I would guess the `netrc` documentation will tell you what sort of errors this function can raise if credentials are invalid, or an internet connection is not available, etc.

Reply:
@brett here is an example: https://www.programcreek.com/python/example/10313/netrc.NetrcParseError

Perhaops we can play a bit with this in the next PR. You will need to modify it as for example an empty file is not an error given that then the DAAC api will ask for username and pasword. I think it should only error out if the file is not with chmod 600. If the credentials are wrong it would erorr out in the DAAC API.